package.json

```diff
@@ -23,11 +23,11 @@
     "wreck": "6.x.x"
   },
   "peerDependencies": {
-    "hapi": ">= 8.x.x"
+    "hapi": ">=8 <10"
```

Comment:
Can you use the verbose syntax instead? I find the terse version much more difficult to read.